lms/views/dashboard/api/assignment.py

```diff
@@ -134,7 +134,6 @@ def assignment(self) -> APIAssignment:
         schema=AssignmentsMetricsSchema,
     )
     def course_assignments_metrics(self) -> APIAssignments:
-        # pylint:disable=too-many-locals
```

Comment:
While pylint has a point, this method could probably use a refactor adding a more local variables triggers the issue in ruff.

Not sure what the discrepancy in the count is, both linters default to 15 as the threshold.